samples/tab-device-permissions/nodejs/src/components/PeoplePicker.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+import * as microsoftTeams from "@microsoft/teams-js";
+import { Card, Flex, Text, Button, CardHeader, CardBody } from '@fluentui/react-northstar'
+
+/**
+ * The 'PeoplePicker' component for 
+ * of your app.
+ */
+const PeoplePicker = () => {
+ const [selectedPeople, setSelectedPeople] = useState<microsoftTeams.people.PeoplePickerResult[]>([]);
+ useEffect(() => {
+  microsoftTeams.initialize()
+ })
+ 
+ // Method to select people using people picker control
+ function selectPeople()
```

Comment:
Could you please format all these code files? 
![image](https://user-images.githubusercontent.com/31851992/125174327-4582f180-e1e2-11eb-8cab-bc567a377842.png)